spec/bolt_server/app_integration_spec.rb

```diff
@@ -135,6 +135,38 @@ def app
       end
     end
 
+    describe "run_script" do
```

Comment:
It seems a little odd to me that we're only testing these endpoints in `app_integration_spec.rb`, rather than `transport_app_spec.rb`, which seems like a natural place for testing changes to that file.

Reply:
I _think_ that this is just a poorly named file. I'm pretty sure the use of the word "integration" is supposed to identify that this test actually sets up the entire service and runs it with some example nodes. `transport_app_spec.rb` seems to only have tests that mock any underlying calls out to nodes, whereas `app_integration_spec.rb` appears to be actually running things on example nodes.

Reply:
When I initially refactored the transport app for SSH and WinRM to accept multiple actions (instead of just run-task), I did the minimum here to make sure we could add new actions quickly - I did not try to reorganize the tests. There just hasn't really been any effort in any direction - reoganization is welcome as far as I'm concerned. I don't think it makes sense right now either.

Reply:
FWIW I think rename the file might make things make a little more sense, but I don't know if we have a ton of time to do much more than that right now with the timeline we are on.

Reply:
I was mostly commenting here to make a note, no rush in making any immediate changes as far as I'm concerned.